tests/plotting/test_examples.py

```diff
@@ -0,0 +1,8 @@
+"""Unit-tests for :mod:`geovista.examples`."""
+
+from geovista.examples import earthquakes
+
+
+def test_earthquakes(verify_image_cache):
+    """Test earthquakes example."""
+    earthquakes.main(off_screen=True, verify_image_cache=verify_image_cache)
```

Comment:
@tkoyama010 Why is it necessary to pass-through the `verify_image_cache` fixture?

Reply:
@bjlittle I'm sorry. The test seems not working properly and I am investigating. At least at this point, we should have an error that `fastparquet` and `pandas` are not installed.